src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLStopImpl.java

```diff
@@ -411,15 +409,7 @@ public DataFetcher<Iterable<NearbyStop>> transfers() {
             .filter(transfer -> maxDistance == null || transfer.getDistanceMeters() < maxDistance)
             .filter(transfer -> transfer.to instanceof Stop)
             .map(transfer ->
-              new NearbyStop(
-                transfer.to,
-                transfer.getDistanceMeters(),
-                transfer.getEdges(),
-                GeometryUtils.concatenateLineStrings(
-                  transfer.getEdges().stream().map(Edge::getGeometry).collect(Collectors.toList())
-                ),
-                null
-              )
+              new NearbyStop(transfer.to, transfer.getDistanceMeters(), transfer.getEdges(), null)
```

Comment:
is there a change in the logic here?

Reply:
No, the geometry was never read from the object, so no need to generate it.